openmdao/core/driver.py

```diff
@@ -79,20 +113,42 @@ def _setup_driver(self, problem):
         self._objs = model.get_objectives(recurse=True)
         self._cons = model.get_constraints(recurse=True)
 
-    def get_design_var_values(self):
+        self._rec_mgr.startup(self)
+        if (self._rec_mgr._recorders):
+            self._model_viewer_data = _get_viewer_data(problem)
+        self._rec_mgr.record_metadata(self)
+
+    def get_design_var_values(self, filter=None):
         """
         Return the design variable values.
 
         This is called to gather the initial design variable state.
 
+        Parameters
+        ----------
```

Comment:
Yes, before we record, we're allowing the users to filter design vars (cons and objs too), using include/excludes.  So before we record, we grab the values of only the ones that they want.  Do you feel like we need to do something with the scaling here before we record?  (We record scaling factors in driver metadata, for what that's worth.)